ui/pages/manufacturers.vue

```diff
@@ -0,0 +1,40 @@
+<template>
```

Comment:
Why is this `manufacturers.vue` and not `manufacturers/index.vue` (like `categories/index.vue`)?

Reply:
Functionally, both are equivalent. But since there is no other subpage of `/manufacturers`, I didn't want to create a folder just for that one `index.vue`.